src/main/kotlin/MockClass.kt

```diff
@@ -0,0 +1,5 @@
+class MockClass {
+    //test only project fails to compile without at least one file in src/main/kotlin
+    //Directory '/Users/anymac/projects/web3j/web3j-unitexample/build/classes/java/main' specified for property 'compileKotlinOutputClasses' does not exist.
+    //https://youtrack.jetbrains.com/issue/KT-31116
+}
```

Comment:
Is it necessary to have this empty class? if we need only the comments maybe can be added to readme file

Reply:
Problem with a gradle build process. 
the issue is https://youtrack.jetbrains.com/issue/KT-31116